### ISSUE TITLE
App Bundle fails to build because of duplicate ic_launcher

### ISSUE THREAD

**Issue by jamieadkins95** [Estimote/Android-Proximity-SDK#91]
## Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * My beacons have Estimote Monitoring enabled (Check it in the Estimote Cloud, or via app)
    * My Android device/devices supports BLE and has Android OS version >= 5.0.0 
    * My Android device/devices have bluetooth enabled
    * My app has Location Permissions granted
    
## Basic information

**Estimote SDK version:** `1.0.3`]

**Android devices affected:** N/A

**Android OS version affected:** N/A

**Beacon hardware version:** N/A

## Description

When building an Android App Bundle I get an exception that says multiple `ic_launcher`s have been declared with different content. I can see that com.estimote.ketchup has a `ic_launcher`. Is this needed?

I can work around this issue by just renaming my `ic_launcher` to something else, but it's not obvious what is happening when this error occurs, so it'd be great to avoid this altogether by deleting your unneeded `ic_launcher`.

**(Optional) Steps to reproduce:**

1. Declare an `ic_launcher` in your app
2. Build an app bundle `/gradlew bundleDebug`

**Expected behavior:** Estimote SDK should not declare a `ic_launcher`

**Actual behavior:**  Estimote SDK declares an `ic_launcher`

## Additional information
```
* What went wrong:
Execution failed for task ':coreapplication:packageDevDebugBundle'.
> java.util.concurrent.ExecutionException: com.android.tools.build.bundletool.exceptions.ValidationException: Modules 'beacons' and 'base' contain entry 'res/mipmap-mdpi-v4/ic_launcher.png' with different content.
```

**Comment by jamieadkins95** [comment]
<img width="799" alt="Screenshot 2019-04-14 at 00 24 24" src="https://user-images.githubusercontent.com/9338082/56086310-4e8e8800-5e4c-11e9-848b-20c1e6b43a51.png">

**Comment by SyedSaadUrRehman-MAD** [comment]
yes this is a severe bug, It occurred to me only while signing the bundle and bundle has 

> <dist:fusing dist:include="false" />

 in the manifest.
In my case i have an icon named ic_date only in the base project, dynamic bundle res folder was empty. the error i got was same, I **RENAMED** the ic_date to ic_cal_date. 
After this change, i have another icon named "ic_time.png", followed same steps and i got my bundle.aab. I deduced that these names of icons are already **TAKEN** and we should not use such names.

**Comment by LokmaneKrizou** [comment]
Hi @jamieadkins95 , have you been able to solve this issue, we have similar issue, two libraries have the same resource name
